### PR TITLE
Added PnP PowerShell commands to Office 365 CDN page

### PR DESCRIPTION
**Pull request by KoenZomers**
Added the steps for enabling and configuring Office 365 CDN using PnP PowerShell

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/use-office-365-cdn-with-spo.md [hunk 1]
```diff
@@ -438,6 +439,289 @@ For more information about this cmdlet, see [Set-SPOTenantCdnEnabled](https://te
...
+
+Now use the **Get-PnPTenantCdnEnabled** cmdlet to retrieve the CDN status settings from the tenant:
+
+``` powershell
+Get-PnPTenantCdnEnabled -CdnType <Public | Private>
+```
+
+The status of the CDN for the specified CdnType will output to the screen.
+
+Use the **Set-PnPTenantCdnEnabled** cmdlet to enable your organization to use the Office 365 CDN. You can enable your organization to use public origins, private origins, or both at once. You can also configure the CDN to skip the setup of default origins when you enable it. You can always add these origins later as described in this topic.
```
Comment (JohanFreelancer9) [inline Enterprise/use-office-365-cdn-with-spo.md]:
```suggestion
Use the **Set-PnPTenantCdnEnabled** cmdlet to enable your organization to use the Office 365 CDN. You can enable your organization to use public origins, private origins, or both. You can also configure the CDN to skip the setup of default origins when you enable it. You can always add these origins later as described in this topic.
```

File: Enterprise/use-office-365-cdn-with-spo.md [hunk 2]
```diff
@@ -47,9 +47,10 @@ To set up the Office 365 CDN for your organization, you follow these basic steps
   + [Determine where you want to store your assets](use-office-365-cdn-with-spo.md#CDNStoreAssets). This location can be a SharePoint site, library or folder and is called an _origin_.
   + [Choose whether each origin should be public or private](use-office-365-cdn-with-spo.md#CDNOriginChoosePublicPrivate). You can add multiple origins of both public and private types.
 
-+ Set up and configure the CDN, using either PowerShell or the SharePoint Online CLI
++ Set up and configure the CDN, using either SharePoint Online Management PowerShell, PnP PowerShell or the Office 365 CLI
```
Comment (JohanFreelancer9) [inline Enterprise/use-office-365-cdn-with-spo.md]:
```suggestion
+ Set up and configure the CDN, using either SharePoint Online Management PowerShell, PnP PowerShell, or the Office 365 CLI
```

Comment (KoenZomers) [reply Enterprise/use-office-365-cdn-with-spo.md]: Is it correct English to put a comma in front of an or?

Comment (KoenZomers) [reply Enterprise/use-office-365-cdn-with-spo.md]: After doing some research on this, I found out that it's both acceptable in correct English. To use it with and without a comma in front of an or. Wouldn't do it myself, but approving this suggestion as it's still okay.

**Comment by KoenZomers** [comment]
@mypil reviewed and ready for merge

**Comment by JohanFreelancer9** [comment]
@KoenZomers Apologies for the delayed response. I do not like it myself, but it is the Oxford comma and in Microsoft documentation it is not optional. It is prescribed by the Microsoft Style guide (https://docs.microsoft.com/en-us/style-guide/punctuation/commas).

**Comment by KoenZomers** [comment]
Thanks for sharing! It was actually a copy paste from the piece on SPO Management Shell in the same article, so you may want to correct it there as well.

**Comment by mypil** [comment]
@get-itips - As discussed, please test and do the PS code review for this PR. Thank you.

**Comment by mypil** [comment]
@kelleyvice-msft - copy/edit and PS code validation review has been done. PR is ready for final review.

Thank you.

cc: @kenwith

**Comment by kenwith** [comment]
@KoenZomers and @get-itips - seems like there are a significant number of conflicts now. Maybe another PR already fixed? Can you please review and let me know the best way to proceed?

**Comment by get-itips** [comment]
Hello @KoenZomers Koen, I tracked down and I identified that you had already created a PR https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/pull/668 with same additions, and it was merged, can you confirm we can close without merging this one ?

**Comment by kenwith** [comment]
Closing this one since #668 resolves. If I should reopen it and we work through the conflicts let me know. Thanks all!

**Comment by KoenZomers** [comment]
Seems like I've been doing the same update twice indeed :) Oops. Okay to close it.

**Pull request closed without merging**